apps/condo/domains/common/schema/fields.js

```diff
@@ -30,7 +31,31 @@ const SENDER_FIELD = {
     },
 }
 
+
+const ADDRESS_META_FIELD = {
+    schemaDoc: 'Property address components',
+    type: Json,
+    isRequired: true,
+    kmigratorOptions: { null: false },
+    hooks: {
+        validateInput: ({ resolvedData, fieldPath, addFieldValidationError }) => {
+            if (!resolvedData.hasOwnProperty(fieldPath)) return addFieldValidationError(`${REQUIRED_NO_VALUE_ERROR}${fieldPath}] Value is required`)
+            const value = resolvedData[fieldPath]
+            if (typeof value !== 'object' || value === null) { return addFieldValidationError(`${JSON_EXPECT_OBJECT_ERROR}${fieldPath}] ${fieldPath} field type error. We expect JSON Object`) }
+            const { dv } = value
+            if (dv === 1) {
```

Comment:
could you please add meta validation

Reply:
It's out of this task scope, I've just extracted already existing declaration of `addressMeta` field.
Since meta-information comes from third-party address suggestion service, which we don't control, a schema may be changed in unpredictable time and saving feature of several entities will be broken.
Do you really have a case, where we must have a value of `addressMeta` field in strict format?

Reply:
I'l be back to this comment in SBERDOMA-879